dashboard/config/scripts/levels/allthethings_curriculum_reference.level

```diff
@@ -6,9 +6,12 @@
   "user_id": 556,
   "properties": {
     "instructions_important": "false",
-    "reference": "/csd/maker_leds/index.html"
+    "reference": "/docs/csd/maker_leds/index.html"
   },
   "published": true,
-  "notes": ""
+  "notes": "",
+  "audit_log": "[{\"changed_at\":\"2018-01-04 10:42:29 -0800\",\"changed\":[\"reference\"],\"changed_by_id\":556,\"changed_by_email\":\"brent@code.org\"},{\"changed_at\":\"2018-01-04 10:55:41 -0800\",\"changed\":[],\"changed_by_id\":556,\"changed_by_email\":\"brent@code.org\"},{\"changed_at\":\"2018-01-04 10:57:16 -0800\",\"changed\":[],\"changed_by_id\":556,\"changed_by_email\":\"brent@code.org\"}]",
+  "level_concept_difficulty": {
+  }
```

Comment:
Intentional?

Reply:
Nope. A bunch of these showed up after I seeded locally, and I meant to clean them all up, but I guess I missed one. Will remove.